### ISSUE TITLE
Add .cs: Content by default

### ISSUE THREAD

**Issue by azz** [azz/vscode-csproj#10]

**Comment by azz** [comment]
Published with version `0.7.0`

**Comment by tegdtna** [comment]
This feature does not work for me by default when installing v0.7.0. I still have to manually add the following setting to get the desired behavior:
```
"csproj.itemType": {
        "*": "Content",
        ".cs": "Compile",
        ".ts": "TypeScriptCompile"
},
```

I am not familiar with vscode extension defaults, but maybe this object needs updating?
https://github.com/azz/vscode-csproj/blob/9ac42aaf92359356bf2cf0c20e2eb987c611a477/package.json#L47-L50

Thanks,
